### ISSUE TITLE
aiohttp v3.8.0 breaks lona

### ISSUE THREAD

**Issue by mwx23** [lona-web-org/lona#175]
`aiohttp` just released a new version that appears to break `lona`.

Pinning the version to `aiohttp==3.7.4` fixes the issues.

Here's the error:

```
MainThread                     DEBUG    22:07:24.994381 lona.middlewares running <lona.middlewares.sessions.LonaSessionMiddleware object at 0x10bcfb3a0>.on_startup(<MiddlewareData(server=<lona.server.LonaServer object at 0x10bcec5e0>)>)
MainThread                     ERROR    22:07:24.994720 lona.middlewares Exception raised while running <bound method LonaSessionMiddleware.on_startup of <lona.middlewares.sessions.LonaSessionMiddleware object at 0x10bcfb3a0>>
  Traceback (most recent call last):
    File "/Users/mwx23/lona/lona/middleware_controller.py", line 146, in _run_middlewares_async
      await hook(data)
    File "/Users/mwx23/lona/lona/middlewares/sessions.py", line 45, in on_startup
      self.generate_session_key = await data.server.run_function_async(
  RuntimeError: Task <Task pending name='Task-1' coro=<_run_app() running at /Users/mwx23/lona/venv/lib/python3.9/site-packages/aiohttp/web.py:321> cb=[_run_until_complete_cb() at /usr/local/Cellar/python@3.9/3.9.7_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py:184]> got Future <Future pending cb=[_chain_future.<locals>._call_check_cancel() at /usr/local/Cellar/python@3.9/3.9.7_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/futures.py:384]> attached to a different loop
```

**Comment by fscherf** [comment]
@mwx23 Thanks for reporting! I noticed that too this morning and i created #174 to fix that. I asked if this is a good solution in the aiohttp discourse https://aio-libs.discourse.group/t/aiohttp-web-run-app-incompatibility-between-aiohttp-3-8-and-aiohttp-3-8/259

**Comment by mwx23** [comment]
@fscherf haha I spent all day debugging this! Was going mad reverting to HEAD on my project and recreating the virtualenv. Glad you're on top of it :)

**Comment by fscherf** [comment]
@mwx23 :D the problem is fixed in 1.7.6
